manifests/pod.yaml

```diff
@@ -1,34 +1,121 @@
-# This pod template is used in the Jenkins pipeline by the Kubernetes plugin
+# This pod template provides the same functionality as pod.yaml, but includes a
+# a configMap to adding a CA-Certifiate to trust (i.e. private Git Repo).
 
 apiVersion: v1
+kind: Template
```

Comment:
can this pod.yaml file actually be an openshift template itself? It's consumed by the kubernetes plugin in the Jenkinsfile, so I'm not sure:

https://github.com/coreos/fedora-coreos-pipeline/blob/fb43e759bd73b7f2c6aade2d66d501687fb42cab/Jenkinsfile#L4